package.json

```diff
@@ -43,7 +43,7 @@
     "d3-format": "^3.0.0",
     "d3-scale": "^3.3.0",
     "d3-scale-chromatic": "^2.0.0",
-    "lodash-es": "^4.17.21",
+    "lodash": "^4.17.21",
```

Comment:
I double-checked: the size of lodash in the output bundle stays exactly the same.